docs/can-guides/commitment/recipes/tinder-like-carousel/Tinder-Like-Carousel.md

```diff
@@ -174,57 +174,48 @@ can.viewModel(document.querySelector("evil-tinder")).profiles.shift()
   },
   ```
 
-  > NOTE Use `.get(0)` to make sure `currentProfile` changes when a is removed from the list.
-
-
-### How to verify it works
-
-Run the following in the `CONSOLE` tab.  The background image should move to the foreground.
-
-```js
-can.viewModel(document.querySelector("evil-tinder")).profiles.shift()
```

Comment:
I removed this section because you don’t have access to `can` or `viewModel` from CodePen’s console. Other than explicitly assigning `window.viewModel = viewModel` in the module, it’d be nice to have another way of verifying this step.

Reply:
componentElement.viewModel works now

Reply:
Updated to show that, thanks @justinbmeyer!